<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>igra</class>
 <widget class="QWidget" name="igra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>igra</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(0,255,255);</string>
  </property>
  <widget class="QPushButton" name="start_gumb">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(128, 110, 93); color:white;border:none;font-size:20px;</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLabel" name="pike">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px</string>
   </property>
   <property name="text">
    <string>PIKE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="izpis_pik">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_gumb">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(128, 110, 93); color:white;border:none;font-size:20px;</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="stanje">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
